src/commands/imageSearch.ts

```diff
@@ -0,0 +1,39 @@
+import Discord from "discord.js";
+import { searchId, googleToken } from "../config.json";
+import fetch from "node-fetch";
+
+/**
+ * Initiates a google image search on specified term and responds with a link to the image
+ * @param {Discord.Messaage} message
+ */
+// Search parameters at https://developers.google.com/custom-search/v1/reference/rest/v1/cse/list
```

Comment:
can you put the comment somewhere else? Not between the documentation message and the function